src/anaconda_auth/actions.py

```diff
@@ -226,11 +226,14 @@ def logout(config: Optional[AnacondaAuthConfig] = None) -> None:
         pass
 
 
-def is_logged_in() -> bool:
+def load_token_info() -> Optional[TokenInfo]:
```

Comment:
I like the idea of a simple function. Also, just FYI, I think a lot of this desired behavior is now captured in the updated implementation of the `create=True` flag in `TokenInfo.load()`. [See here](https://github.com/anaconda/anaconda-auth/blob/main/src/anaconda_auth/token.py#L248-L282).

In other words, if you do `TokenInfo.load(create=True)`, it will return a `TokenInfo` object associated with the default `domain` from `AnacondaAuthConfig().domain` (by default, unless you explicitly pass a domain in. That object, if it was created and not loaded from the keyring, will have `token_info.api_key is None`.

But I'm totally open for feedback and to add a convenience wrapper as a simple function, if the `@classmethod` API is too confusing.

Does this align with what you are thinking?

Reply:
Thanks for your input, Matt! TokenInfo.load() does the job and meets all our needs. The idea for the PR actually came after using the CLI and noticing there was a straightforward way to retrieve the token. When I looked for the equivalent in the Python interface, I couldn’t find something at the same "level" of `login` and `logout`.

The wrapper simply makes it more convenient and intuitive, but it’s more of a nice-to-have rather than a necessity. Appreciate the discussion!